app/Core/Language.php

```diff
@@ -80,6 +80,11 @@ class Language
      */
     public Theme $themeCore;
 
+    /**
+     * @var Apirequest $apiRequest
+     */
+    public Apirequest $apiRequest;
```

Comment:
Can you update the name to be ApiRequest (with capital R) the acceptance tests fail because it can't find the file (linux systems are case sensitive)

Reply:
It is now fixed. Sorry bout that.

Reply:
No worries, thanks